### ISSUE TITLE
Change of basis matrix ordering issue

### ISSUE THREAD

**Issue by bzrry** [pim-book/programmers-introduction-to-mathematics#51]
On page 192 of the first edition of the book, it says “Let A be the matrix representation of a linear map f, written with respect to the standard basis. Let U be a change of basis matrix. That is, the columns of U are the new basis vectors, and if we were to write f with respect to the new basis, its matrix would be B = UAU⁻¹.”

In fact, a matrix consisting of the basis vectors of U converts a vector from the U basis to the standard basis, so B should equal U⁻¹AU.

**Comment by j2kun** [comment]
Yes, that is a mistake! Page 192 can safely switch U and U^{-1} throughout the theorem and proof statement
